system/gatekeeper/tests/fixtures/vulnerable-images/pod-low.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: dummy
+status:
+  containerStatuses:
+    - image: keppel.example.com/vulnerability:low
+      imageID: docker-pullable://keppel.example.com/vulnerability:low@sha256:0000000000000000000000000000000000000000000000000000000000000000
+      name: dummy
+      ready: true
+      restartCount: 0
```

Comment:
As a test case for the aforementioned problem, please combine some of the test cases into a single Pod with multiple containers, to prove that the policy actually checks all of the containers in it, not just one of them.